src/components/Chip/Chip.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import styles from './Chip.scss';
+
+const { block } = bem({
+    name: 'Chip',
+    classnames: styles
+});
+
+const Chip = props => {
+    const { children, title, ...rest } = props;
+
+    return (
+        <span title={title} {...rest} {...block(props)}>
+            {children}
+        </span>
+    );
+};
+
+Chip.displayName = 'Chip';
+
+Chip.propTypes = {
+    /** The content of the chip: it is expected to be short text possibly accompanied by an icon */
+    children: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+    /** Text for title chip that shows up when it is hovered for a few seconds */
+    title: PropTypes.string
```

Comment:
Wouldn't it make more sense for this to be an 'undocumented prop' that just comes from `...rest`? Eventually the idea is for those chips to have 'real' tooltips rather than html ones anyway

Reply:
I think it is better to have it documented, exactly so that it can be easily changed to use a tooltip if we want. Then we can just change the internal implementation and the code in JF, Search doesn't have to be changed. So title will be used for the tooltip. If you think that prop should have an other name, we can change it already so that it fills more universal. Any suggestions?

Reply:
I think eventually the tooltip would not be managed by those components themselves (i.e. there would not be a prop for giving specific components a tooltip). Probably we shouldn't even bother with making tooltips of our own (only the styling). Most tooltip libraries (e.g. [react-tooltip](https://github.com/wwayne/react-tooltip#usage)) use attributes that you have to add to any component or element you want to give a tooltip.